politeiawww/proposals_test.go

```diff
@@ -472,34 +468,39 @@ func TestValidateProposal(t *testing.T) {
 				ErrorCode: www.ErrorStatusProposalMissingFiles,
```

Comment:
This should probably be changes to something like `ErrorStatusNoIndexFile` to be more descriptive.  Also, you need to rework these tests.  The test "bad md filename" no longer makes sense.  Markdown files can have whatever name they want as long as an `index.md` file exists.